index.rst

```diff
@@ -6,21 +6,44 @@
 Welcome to Open Simulation Interface's documentation!
 =====================================================
 
-The Project
+The Vision
 -----------
+
 The Open Simulation Interface (OSI) is a generic interface for the environmental perception of automated driving functions in virtual scenarios.
```

Comment:
I think this sentence is not so suitable anymore. Would prefer the following: "The Open Simulation Interface (OSI) is an interface specification between models and components of a distributed simulation. A strong focus is on the environmental perception part of automated driving functions."

Reply:
- [ ] Rewrite intial sentence.